src/inc/quic_driver_connection.h

```diff
@@ -0,0 +1,376 @@
+/*++
+
+    Copyright (c) Microsoft Corporation.
+    Licensed under the MIT License.
```

Comment:
I'm not a big fan of the file name. Let's call this quic_driver_helpers.h. Also, please add a bit of an abstract at the top.